code/ui/manager/src/components/panel/Panel.tsx

```diff
@@ -60,6 +60,23 @@ export const AddonPanel = React.memo<{
         {...(selectedPanel ? { selected: selectedPanel } : {})}
         menuName="Addons"
         actions={actions}
+        showToolsWhenEmpty
+        customEmptyState={
+          <Placeholder>
+            <Fragment key="title">
+              <P>Storybook add-ons</P>
+            </Fragment>
```

Comment:
the fragment seems unnecessary here?

```suggestion
            <P>Storybook add-ons</P>
```